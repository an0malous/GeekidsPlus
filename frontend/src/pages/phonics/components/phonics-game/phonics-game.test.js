import React from 'react'
import renderer from 'react-test-renderer';
import PhonicsGame from './phonics-game.component.jsx';

describe('AlphabetCard Component', () => {
   let component;

   beforeEach(() => {
     component = renderer
     .create(<PhonicsGame />)
     .toJSON()
   });
   
   it('Renders', () => {
      expect(component).toMatchSnapshot()
   })
});

