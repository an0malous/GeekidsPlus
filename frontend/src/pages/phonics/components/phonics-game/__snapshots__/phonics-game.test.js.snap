// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlphabetCard Component Renders 1`] = `undefined`;
